dev/com.ibm.ws.concurrent.persistent_fat/fat/src/com/ibm/ws/concurrent/persistent/fat/PersistentExecutorTest.java

```diff
@@ -38,8 +39,8 @@
     @Server("com.ibm.ws.concurrent.persistent.fat")
     @TestServlet(servlet = SchedulerFATServlet.class, path = APP_NAME)
     public static LibertyServer server;
-    
-    public static final JdbcDatabaseContainer<?> testContainer = FATSuite.testContainer;
+
+    public static final JdbcDatabaseContainer<?> testContainer = DatabaseContainerFactory.create();
```

Comment:
You can add @ClassRule here and remove `testContainer.start()` and stop later on.